sdl.c

```diff
@@ -522,6 +530,36 @@ static int sdl_poll_one(struct sdl_ctx *c, SDL_Event *evt, uint16_t min,
 	return NOTHING;
 }
 
+static SDL_Point calc_point_from_buf_offset(const struct sdl_ctx *c, const int offset)
```

Comment:
This return-by-value is fine, SDL_Point is two `int`s so it can be returned in a pair of registers, and it gets inlined anyway.